paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -179,6 +180,68 @@ class PaymentSheet internal constructor(
         IntentConfirmationInterceptor.createIntentCallback = createIntentCallback
     }
 
+    /**
+     * Builder to add optional callbacks to [PaymentSheet].
+     *
+     * @param resultCallback Called with the result of the payment after [PaymentSheet] is dismissed.
+     */
+    class Builder(internal val resultCallback: PaymentSheetResultCallback) {
+        internal var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler? = null
+            private set
+        internal var createIntentCallback: CreateIntentCallback? = null
+            private set
+
+        /**
+         * @param handler Called when a user confirms payment for an external payment method.
+         */
+        fun externalPaymentMethodConfirmHandler(handler: ExternalPaymentMethodConfirmHandler) = apply {
+            externalPaymentMethodConfirmHandler = handler
+        }
+
+        /**
+         * @param intentCallback Called when the customer confirms the payment or setup.
+         */
+        fun createIntentCallback(intentCallback: CreateIntentCallback) = apply {
+            createIntentCallback = intentCallback
+        }
+
+        /**
+         * Returns a [PaymentSheet] and initializes callback handlers if respective callbacks are set
```

Comment:
Can we make all of these doc strings sentences (end in a period)?

Reply:
Updated